minecraft_servers/cidon/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "server_name": "cidon",
+  "nice_name": "Cidon",
+  "direct_ip": "cidon.de",
+  "server_wildcards": [
+    "%.cidon.de"
+  ],
```

Comment:
Is it really a wildcard?
<img width="1049" alt="image" src="https://github.com/LabyMod/server-media/assets/37186532/d1d18aa6-257c-42a3-ba8b-043f510488d1">

Reply:
I've removed the wildcards with https://github.com/LabyMod/server-media/pull/983/commits/00fb77febfa95ba10bd34ade623c7c2c6079b8f3